### PR TITLE
Update to 1.0.14 and refactor

### LINKED ISSUE #4: Accents doesn't work
Accents as produced by an es-traditional layout doesn't work in the text input of the chat and search. It shows the letter without the accent (`a` instead of `á`) and checking on the phone it doesn't show the accent on mobile either.  Other characters like the `ñ` show correctly.

Edit: doesn't happen with the official Linux version (non snap) version.

### PR DESCRIPTION
**Pull request by flexiondotorg**
This pull request introduces the following changes:

  * Update to 1.0.14
  * Update `summary:` and `description:` based on the content from the Telegram website.
  * Add `libasound2` and `libpulse0` to `stage-packages:` to correctly bring up audio.
  * Add `network-manager` interface which requires manual connection. Convenience script `connect.sh` included.
  * Migrate to `snap/` directory.
  * Drop `telegram.sh` wrapper in favour of `desktop-glib-helper`.
    * The desktop helper provides all the directory creation/chmodding the old wrapper did but also helps 
silence some runtime warnings and sets the paths correctly for PulseAudio.
    * Also fixes #4.
  * Use `environment:` stanza for setting additional environment variables specific to this snap.

### CODE REVIEW

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+name: telegram-sergiusens
+version: 1.0.14
```
Comment (sergiusens) [inline snap/snapcraft.yaml:2]: ah, I forgot to push an update for this!

File: snap/snapcraft.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+summary: A cloud-based messaging app with a focus on security and speed.
```
Comment (sergiusens) [inline snap/snapcraft.yaml:3]: interesting new summary :-)

**Review by sergiusens — APPROVED** [review]
thanks for the contribution!

I only worry a bit about `network-manager` but that I guess manual connection makes it ok.

**Comment by juanjux** [comment]
Still not working, I removed the normal telegram and the snap and reinstalled the snap just to be sure (it installed version 1.0.14), accents still doesn't work (but "ñ" and other non composed characters do) and entering the cloud password on the first login page shows squares instead of the usual big-dots, it you need my help to debug the problem just tell me what I've to do, I'm testing under KDE with the spanish traditional keyboard layout.

**Comment by emanuelserpa** [comment]
Strange, still not working here. And in Brave accents works fine.

**Pull request merged**